### PR TITLE
Milestone 04 : controllers

### PR DESCRIPTION
**Pull request by FaranoshAmini**
## In this pull request, I have implemented the following:

- [x]  For each URL from the wireframes, I created a `controller`, an action in the correct `controller`, `route`, and a `view` file.
- [x] View files included only placeholders written in plain HTML.
- [x] Made sure that I added code to handle only the 4 required URLs from the wireframes.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:id/", to: "users#show"
+  get "/users/:user_id/posts/:id", to: "posts#show"
+  get "/users/:user_id/posts", to: "posts#index"
```
Comment (TedLivist) [inline config/routes.rb]: - [x] **[OPTIONAL]** I think we can improve the code here by using the alternative `resources` helper as provided by Rails.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @FaranoshAmini,

✔️ Routes, controllers and views are well created

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (FaranoshAmini) [reply config/routes.rb]: @TedLivist Thanks for the observation. I will improve it

**Comment by FaranoshAmini** [comment]
@TedLivist Thanks for your kind review and feedback.

**Pull request merged**